config/application.rb

```diff
@@ -15,6 +15,11 @@
 # you've limited to :test, :development, or :production.
 Bundler.require(*Rails.groups)
 
+# Require all Engines inside the `packages` directory
+Dir.glob("lib/**/engine.rb", base: Rails.root).each do |path|
```

Comment:
I wonder whether we should have all of our engines in `lib/engines/`, just so we're clearer about what these things are. We should probs change the commit title here too.

Reply:
have made this change now to `lib/engines`